rules/catch-error-name.js

```diff
@@ -69,6 +69,13 @@ const create = context => {
 		const scope = context.getScope();
 		const variable = findVariable(scope, node);
 
+		// This was reported https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1075#issuecomment-768072967
+		// But can't reproduce, just ignore this case
```

Comment:
Does this still apply? I noticed you added some tests.

Reply:
I don't know, the smoke test failed 3days ago, I can't reproduce.  Very strange,  same thing happen few days ago https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1045/files#diff-10f1d5918272c29964d48236585e1df3b35709537364651b22f63e9a7e21a3c7R174